app/logic/Calendar/Invitation.ts

```diff
@@ -1,20 +1,24 @@
 export type iCalMethod = "CANCEL" | "REQUEST" | "REPLY";
 
 /**
- * For an inbox item that represents a scheduling message, the type of message:
- * Accepted/Tentative/Declined responses, invitations, or cancellations.
+ * For an inbox item that represents a scheduling message, the type of message.
  */
-export enum Scheduling {
+export enum InvitationMessage {
   None = 0,
-  Accepted = 1,
-  Tentative = 2,
-  Declined = 3,
-  Request = 4,
-  Cancellation = 5,
+  Invitation = 4, // METHOD:REQUEST, an incoming invitation
+  CancelledEvent = 5, // METHOD:CANCEL, a follow-up cancellation
+  ParticipantReply = 6, // METHOD:REPLY, a reply to your outgoing invitation
 }
 
-/* Note: These are EWS/OWA names and ActiveSync values. */
-export enum ResponseType {
+/**
+ * For meetings only, these values indicate the status of either:
+ * - For any given participant, their participation in an event
+ * - The user's own participation in an event
+ *   In particular, this identifies whether the user organised this meeting.
+ *
+ * Note: These are EWS/OWA names and ActiveSync values.
+ */
+export enum InvitationResponse {
   Unknown = 0,
   Organizer = 1,
```

Comment:
```suggestion
  /** Our user is the organizer of this event.
   Signals that this is an outgoing invitation.
   For outgoing invitations only. */
  Organizer = 1,
```

Reply:
As a status on an event, this shows that you were the organiser. (Your participant object will also have the Organiser status.)

For meetings organised by someone else, this is set on the organising attendee, unless the meeting was cancelled.

Reply:
OK, can you please update the JSDoc definition?

If possible, I'd prefer to have 2 enums: one for incoming invitations, and one for outgoing invitations. The code should not be shared between them, so the enum isn't shared, either. Having a separate enum would make the cases clearer.

--- 
Also, please add JSDoc to `Accept` and `Decline` below.

Reply:
I don't understand.

Tom uses Mustang to invite Harry to a meeting. Harry uses Mustang to accept the invitation. This means that both Tom and Harry have a meeting where Tom is the organiser and Harry is an accepted invitee. Why would they use different enums?

Reply:
You're talking about Accept/Decline now. I was referring to the Organizer state that [you explained above](https://github.com/mustang-im/mustang/pull/570#discussion_r2052697983).

But if it makes more sense, you can keep them in the same enum. Just please document for `Organizer` what you said in your previous comment, what it means for incoming and outgoing invitations.